docs/content/preview/yugabyte-voyager/install-yb-voyager.md

```diff
@@ -107,6 +107,10 @@ Install yb-voyager on a machine which satisfies the [Prerequisites](#prerequisit
   </div>
 </div>
 
+### Collect diagnostics
```

Comment:
Does diagnostics deserve its own top level page, or should all the info just go here